build/openresty/patches/ngx_lua-0.10.25_01-dyn_upstream_keepalive.patch

```diff
@@ -1,8 +1,33 @@
+diff --git a/bundle/nginx-1.21.4/src/http/ngx_http_upstream.c b/bundle/nginx-1.21.4/src/http/ngx_http_upstream.c
+index b07e564..68c9b05 100644
+--- a/bundle/nginx-1.21.4/src/http/ngx_http_upstream.c
++++ b/bundle/nginx-1.21.4/src/http/ngx_http_upstream.c
+@@ -4304,6 +4304,7 @@ ngx_http_upstream_next(ngx_http_request_t *r, ngx_http_upstream_t *u,
+     if (u->peer.cached && ft_type == NGX_HTTP_UPSTREAM_FT_ERROR) {
+         /* TODO: inform balancer instead */
```

Comment:
We can delete this line now, right? If I understand you correctly, this PR has already implemented this piece.

Reply:
it can be deleted, but it is not necessary, which is also part of the Nginx code, not OpenResty or ngx_lua_dyn_upstream_keepalive patch